Makefile

```diff
@@ -1,85 +1,42 @@
-# Copyright 2021 NVIDIA
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# Package related
-BINARY_NAME=network-operator
-PACKAGE=network-operator
-ORG_PATH=github.com/Mellanox
-REPO_PATH=$(ORG_PATH)/$(PACKAGE)
-CHART_PATH=$(CURDIR)/deployment/$(PACKAGE)
-GOPATH?=$(CURDIR)/.gopath
-GOBIN =$(CURDIR)/bin
-TOOLSDIR=$(CURDIR)/bin
-BUILDDIR=$(CURDIR)/build/_output
-BASE=$(GOPATH)/src/$(REPO_PATH)
-GOFILES=$(shell find . -name "*.go" | grep -vE "(\/vendor\/)|(_test.go)")
-PKGS=$(or $(PKG),$(shell cd $(BASE) && env GOPATH=$(GOPATH) $(GO) list ./... | grep -v "^$(PACKAGE)/vendor/"))
-TESTPKGS = $(shell env GOPATH=$(GOPATH) $(GO) list -f '{{ if or .TestGoFiles .XTestGoFiles }}{{ .ImportPath }}{{ end }}' $(PKGS))
-
-export GOPATH
-export GOBIN
-
-# Version
-VERSION?=master
-DATE=`date -Iseconds`
-COMMIT?=`git rev-parse --verify HEAD`
-LDFLAGS="-X github.com/Mellanox/network-operator/version.Version=$(VERSION) -X github.com/Mellanox/network-operator/version.Commit=$(COMMIT) -X github.com/Mellanox/network-operator/version.Date=$(DATE)"
-
-# Docker
-IMAGE_BUILDER?=@docker
-IMAGEDIR=$(BASE)/images
-DOCKERFILE?=$(CURDIR)/Dockerfile
-TAG?=mellanox/network-operator
-IMAGE_BUILD_OPTS?=
-# Accept proxy settings for docker
-# To pass proxy for Docker invoke it as 'make image HTTP_POXY=http://192.168.0.1:8080'
-DOCKERARGS=
-ifdef HTTP_PROXY
-	DOCKERARGS += --build-arg http_proxy=$(HTTP_PROXY)
+# VERSION defines the project version for the bundle.
```

Comment:
Hey @adrianchiris,

Indeed, the Makefile has been auto-generated for many releases now. 
I'll try to bring some of that back. Not sure why one would specifically want to build with docker.

Reply:
OK from the list above here is what is missing (*lint):

hadolint
helm-lint
lint (golangci) 

these are covered by github actions you addded. however to run them locally, one would need to install and run each command instead of invoking the Makefile target as it was before.

can you elaborate on why you chose to omit those ?